app/server/appsmith-plugins/graphqlPlugin/src/test/java/com/external/plugins/GraphQLPluginTest.java

```diff
@@ -115,15 +129,16 @@ private ActionConfiguration getDefaultActionConfiguration() {
         ActionConfiguration actionConfig = new ActionConfiguration();
         actionConfig.setHeaders(List.of(new Property("content-type", "application/json")));
         actionConfig.setHttpMethod(HttpMethod.POST);
-        String requestBody = "query {\n" +
-                "\tallPosts(first: 1) {\n" +
-                "\t\tnodes {\n" +
-                "\t\t\tid\n" +
-                "\t\t}\n" +
-                "\t}\n" +
-                "}";
+        String requestBody = """
+                query {
+                \tallPosts(first: 1) {
+                \t\tnodes {
+                \t\t\tid
+                \t\t}
+                \t}
+                }""";
```

Comment:
If we are converting these to multi-line strings, let's do away with the `\t` as well? Makes it more readable I think.

```suggestion
        String requestBody = """
                query {
                  allPosts(first: 1) {
                    nodes {
                      id
                    }
                  }
                }""";
```